literature.bib

```diff
@@ -5745,6 +5936,30 @@ @inproceedings{RAK+:VaMoS13
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2012 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{KKOS:ICSM12,
+	author = {Timo {Kehrer} and Udo {Kelter} and Manuel {Ohrndorf} and Tim {Sollbach}},
+	booktitle = ICSM,
+	title = {{Understanding Model Evolution Through Semantically Lifting Model Differences With SiLift}},
+	year = 2012,
+	pages = {638--641},
+	pb-tags = {semantic lifting},
+	keywords = {graph theory;software engineering;software tools;model evolution;model differences;SiLift;model-based software development;model-based system;graph operations;abstract syntax graph;generic tool environment;EMF-based models;user-level edit operations;Unified modeling language;Adaptation models;Semantics;Engines;Conferences;Software;Software engineering;model comparison;model difference;semantic lifting;difference presentation},
+	doi = {10.1109/ICSM.2012.6405342}
+}
+
+@inproceedings{FKLW:ICSM12,
+	author = {Ameni {ben Fadhel} and Marouane {Kessentini} and Philip {Langer} and Manuel {Wimmer}},
+	booktitle = ICSM,
+	title = {{Search-Based Detection of High-Level Model Changes}},
+	year = 2012,
+	volume = {},
```

Comment:
why to have those 3 empty tags?

Reply:
publisher missing and no comment added

Reply:
Removed empty tags; Added publisher and address to both of these entries.